gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -18,6 +18,67 @@
     <module name="NewlineAtEndOfFile"> <!-- Java Style Guide: Line ending: LF -->
         <property name="lineSeparator" value="lf"/>
     </module>
+    <!-- Inclusive Code checks: https://wiki.palantir.tech/display/DIVERSITY/Inclusive+Language+Glossary -->
+    <module name="RegexpSingleline"> 
+        <property name="format" value="ablebodied"/>
+        <property name="message" value="To convey the same idea and be more inclusive, consider nondisabled instead of ablebodied."/>
+    </module>
+    <module name="RegexpSingleline"> 
+        <property name="format" value="blackout"/>
+        <property name="message" value="To convey the same idea and be more inclusive, consider downtime instead of blackout."/>
+    </module>
+    <module name="RegexpSingleline"> 
+        <property name="format" value="blacklist"/>
+        <property name="message" value="To convey the same idea and be more inclusive, consider denylist instead of blacklist."/>
```

Comment:
I like the wording and in general appreciate the relaxed tone, but I think we can take a stronger stance than "consider" here. "Please use ", perhaps?

I don't want to bikeshed on replacements, but we may be best served by providing a list of alternatives -- I tend toward "banned" or "disallowed" rather than "denylist", but it depends on the context.

Reply:
cool, I can offer a list! i agree that it's hard to provide a single correct replacement without context